src/core/files.py

```diff
@@ -248,7 +248,7 @@ def handle_multiple_email_files(request_files, file_owner):
 
     Args:
         request_files (list): in-memory files from a Django http response.
-        file_owner(django.contrib.auth.models.User): the owner of the saved files.
+        file_owner(django.contrib.auth.models.User): owner of the saved files.
```

Comment:
Should there be a space between `file_owner` and `(django.contrib.auth.models.User)` ?

Reply:
Correct